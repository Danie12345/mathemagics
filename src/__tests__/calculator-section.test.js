import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';

const CalculatorSection = require('../components/calculator-section/CalculatorSection.js').default;

test('loads the container with everything', () => {
  const tree = renderer.create(<CalculatorSection />).toJSON();
  expect(tree).toMatchSnapshot();
})