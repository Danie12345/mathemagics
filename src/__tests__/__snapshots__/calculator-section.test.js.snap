// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads the calculator-section with everything 1`] = `
<div
  className="section calculatorSection"
>
  <p
    className="description"
  >
    Use the very awesome calculator!
  </p>
  <div
    className="calculator"
  >
    <div
      className="result"
    />
    <div
      className="sections"
    >
      <div
        className="left"
      >
        <div
          className="appliers"
        >
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            %
          </button>
        </div>
        <div
          className="operands"
        >
          <div
            className="numbers"
          >
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              1
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              2
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              3
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              4
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              5
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              6
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              7
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              8
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              9
            </button>
          </div>
          <div
            className="pivots"
          >
            <button
              className="button x2"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              0
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              .
            </button>
          </div>
        </div>
      </div>
      <div
        className="right"
      >
        <div
          className="operators"
          type="button"
        >
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            ÷
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            ×
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            -
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            +
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`loads the container with everything 1`] = `
<div
  className="section calculatorSection"
>
  <p
    className="description"
  >
    Use the very awesome calculator!
  </p>
  <div
    className="calculator"
  >
    <div
      className="result"
    />
    <div
      className="sections"
    >
      <div
        className="left"
      >
        <div
          className="appliers"
        >
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            %
          </button>
        </div>
        <div
          className="operands"
        >
          <div
            className="numbers"
          >
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              1
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              2
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              3
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              4
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              5
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              6
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              7
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              8
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              9
            </button>
          </div>
          <div
            className="pivots"
          >
            <button
              className="button x2"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              0
            </button>
            <button
              className="button"
              onClick={[Function]}
              onKeyDown={[Function]}
              type="button"
            >
              .
            </button>
          </div>
        </div>
      </div>
      <div
        className="right"
      >
        <div
          className="operators"
          type="button"
        >
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            ÷
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            ×
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            -
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            +
          </button>
          <button
            className="button"
            onClick={[Function]}
            onKeyDown={[Function]}
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
